inference-engine/src/transformations/include/transformations/op_conversions/simplify_ctc_greedy_decoder.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+
+#include <transformations_visibility.hpp>
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API SimplifyCTCGreedyDecoder;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief SimplifyCTCGreedyDecoder converts v6:CTCGreedyDecoderSeqLen into v0::CTCGreedyDecoder.
+ *
+ * Here are steps that transformation performs:
```

Comment:
I can see this steps from the transformation code. 
The purpose of transformation description is to provide meaningful information about replacement. So I expect to see description which explains why CTCGreedyDecoderSeqLen  can be transformed to CTCGreedyDecoder and which constraints it has.
More detailed explanation like you did you need to put into transformation code.